packages/paste-website/src/pages/components/button-group/index.mdx

```diff
@@ -0,0 +1,252 @@
+---
+title: Button Group
+package: '@twilio-paste/button-group'
+description: A Button Group is a set of related Buttons
+slug: /components/button-group/
+---
+
+import {graphql} from 'gatsby';
+import {Button} from '@twilio-paste/button';
+import {ButtonGroup} from '@twilio-paste/button-group';
+import {PopoverContainer, PopoverButton, Popover} from '@twilio-paste/popover';
+import {Heading} from '@twilio-paste/heading';
+import {Separator} from '@twilio-paste/separator';
+import {Text} from '@twilio-paste/text';
+import {CheckboxCheckIcon} from '@twilio-paste/icons/esm/CheckboxCheckIcon';
+import {PlusIcon} from '@twilio-paste/icons/esm/PlusIcon';
+import {UnderlineIcon} from '@twilio-paste/icons/esm/UnderlineIcon';
+import {ItalicIcon} from '@twilio-paste/icons/esm/ItalicIcon';
+import {BoldIcon} from '@twilio-paste/icons/esm/BoldIcon';
+
+import {SidebarCategoryRoutes} from '../../../constants';
+import {
+  BasicExample,
+  UnattachedExample,
+  DisabledExample,
+  IconExample,
+} from '../../../component-examples/ButtonGroupExamples';
+
+export const pageQuery = graphql`
+  {
+    allPasteComponent(filter: {name: {eq: "@twilio-paste/button-group"}}) {
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(frontmatter: {slug: {eq: "/components/button-group/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+    allAirtable(filter: {data: {Feature: {eq: "Button Group"}}}) {
+      edges {
+        node {
+          data {
+            Documentation
+            Figma
+            Design_committee_review
+            Engineer_committee_review
+            Code
+            status
+          }
+        }
+      }
+    }
+  }
+`;
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/button-group"
+  storybookUrl="/?path=/story/components-button-group--attached"
+  data={props.data}
+  description={props.pageContext.frontmatter.description}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+<LivePreview
+  noInline
+  scope={{
+    ButtonGroup,
+    Button,
+    PlusIcon,
+    CheckboxCheckIcon,
+    PopoverContainer,
+    PopoverButton,
+    Popover,
+    Heading,
+    Separator,
+    Text,
+  }}
+  language="jsx"
+>
+  {BasicExample}
+</LivePreview>
+
+## Guidelines
+
+A Button Group is a set of 2-6 buttons. It is often used to visually group related buttons together. The buttons should be independent from each other, i.e. clicking one button does not change the state of another button.
```

Comment:
Instead of "related buttons" could this say "Related actions"?

Reply:
Yeah!

Reply:
```suggestion
A Button Group is a way to bring a set of 2-6 buttons together, often visually, that have related actions. The buttons should be independent from each other, i.e. clicking one button does not change the state of another button.
```